src/Pages/2DPrinting/App.js

```diff
@@ -16,180 +11,149 @@ import {
+        }}
+        labelIdle="Drag & Drop or Touch Here <br />
+
+            <svg aria-hidden='true' viewBox='0 0 512 512' width='40%'>
+              <path
+                d='M399.95 160h-287.9C76.824 160 48 188.803 48
+              224v138.667h79.899V448H384.1v-85.333H464V224c0-35.197-28.825-64-64.05-64zM352
+              416H160V288h192v128zm32.101-352H127.899v80H384.1V64z'
+              />
+            </svg>"
+      />
+      <Button
+        color='primary'
+        className='continue'
+        onClick={() => setModal(!modal)}
+      >
+        continue
+      </Button>
+      <Modal
+        isOpen={modal}
+        toggle={() => setModal(!modal)}
+        className={props.className}
+        onClosed={closeAll ? toggle : undefined}
+      >
+        <ModalHeader toggle={() => setModal(!modal)}>
+          Software & Computer Engineering Society
+        </ModalHeader>
+        <ModalBody>
+          <br />
+          <FormGroup>
+            <legend for='exampleSelect'>Number of copies</legend>
+            <Input type='select' name='select' id='exampleSelect'>
+              <option>1</option>
+              <option>2</option>
+              <option>3</option>
+              <option>4</option>
+              <option>5</option>
+            </Input>
+            <legend>Type of Print</legend>
+            <FormGroup check>
+              <Label check>
+                <Input type='radio' name='radio1' /> Front
+              </Label>
             </FormGroup>
-            <Label> Note: All prints are black ink only</Label>
-          </ModalBody>
-          <ModalFooter>
-            {/* <Button color="primary" onClick={this.toggle}>Do Something</Button>{' '}
-              <Button color="secondary" onClick={this.toggle}>Cancel</Button> */}
-            <Button color='success' onClick={this.handleToggleNested}>
-              Print!
-            </Button>
-            <Modal
-              isOpen={this.state.nestedModal}
-              toggle={this.toggleNested}
-              onClosed={this.state.closeAll ? this.toggle : undefined}
+            <FormGroup check>
+              <Label check>
+                <Input type='radio' name='radio1' />{' '}
+                {/* add disabled if in need of option disablement */}
+                Front & Back
+              </Label>
+            </FormGroup>
+          </FormGroup>
+          <Label> Note: All prints are black ink only</Label>
+        </ModalBody>
+        <ModalFooter>
+          <Button color='danger' onClick={() => setModal(!modal)}>
+            Back
+          </Button>
+          <Button
+            color='success'
+            onClick={() => {
+              setNestedModal(!nestedModal)
+              setCloseAll(false)
+            }}
+          >
+            Print!
+          </Button>
+          <Modal
+            isOpen={nestedModal}
+            toggle={() => {
+              setNestedModal(!nestedModal)
```

Comment:
earlier we discussed trying to close all the modals at once. when we hit `Print!`, to close all modals the `onClick` function could be 

```js
onClick={() => {
  setNestedModal(false)
  setModal(false)
}}
```
this will make the outer modal close too.